README.md

```diff
@@ -33,620 +33,677 @@ If you want to build the CLI using the source code, please, take into account th
+5aba066c425b5d0c64000002   5aba04be425b5d0c16000000
+```
+
+##### Create our Joomla Server
+
+```bash
+$ concerto cloud servers create --name joomla-node1 --workspace_id 5aabb7521de0240abb00000e --template_id 5af9aab5ea7a1a000d00000e --server_plan_id 5aac0c04348f190b3e001186 --cloud_account_id 5aabb7531de0240abb000024
+ID:                 5af9acedea7a1a000d000012
+NAME:               joomla-node1
+FQDN:
+STATE:              commissioning
+PUBLIC_IP:
+WORKSPACE_ID:       5aabb7521de0240abb00000e
+TEMPLATE_ID:        5af9aab5ea7a1a000d00000e
+SERVER_PLAN_ID:     5aac0c04348f190b3e001186
+CLOUD_ACCOUNT_ID:   5aabb7531de0240abb000024
+SSH_PROFILE_ID:     5aabb7521de0240abb00000d
 ```
 
-To create a kubernetes master for this cluster, we execute this command
-```
-$ concerto nodes create --cluster k8sconcerto --plan 55b0916d10c0ecc35100040e
-```
-The kubernetes master will be commissioned and start booting. Give it some time for the cloud provider to acquire the IP, and issue this command
-```
-$ concerto nodes list
-CLUSTER           MASTER         ID                         NAME                        FQDN                                             IP               STATE
-k8sconcerto       *              56420d894579c0932e00000c   k8sconcerto-master-01       k8sconcerto-master-01.ingrammicro.concerto.io       178.62.111.185   bootstrapping - 10%
-```
-Our master is booting, wait untill it becomes operational and launch a new node. Concerto will instance first node as a master, and subsequent nodes as kubernetes slaves.
-```
-$ $ concerto nodes create --cluster k8sconcerto --plan 55b0916d10c0ecc35100040e
-```
-Now we have a kubernetes master and a slave. If you need to manage kubernetes nodes or containers use `concerto nodes` subcommands.
-If you need to manage the kubernetes cluster, use `concerto cluster` subcommands.
-```
-$ concerto cluster kubectl --cluster k8sconcerto run single-container-pod --image=nginx
-CONTROLLER             CONTAINER(S)           IMAGE(S)   SELECTOR                   REPLICAS
-single-container-pod   single-container-pod   nginx      run=single-container-pod   1
+And finally boot it
 
-concerto cluster kubectl --cluster k8sconcerto get pods
-NAME                         READY     STATUS    RESTARTS   AGE
-single-container-pod-0uhfr   0/1       Pending   0          56s
+```bash
+$ concerto cloud servers boot --id 5af9acedea7a1a000d000012
+ID:                 5af9acedea7a1a000d000012
+NAME:               joomla-node1
+FQDN:
+STATE:              booting
+PUBLIC_IP:
+WORKSPACE_ID:       5aabb7521de0240abb00000e
+TEMPLATE_ID:        5af9aab5ea7a1a000d00000e
+SERVER_PLAN_ID:     5aac0c04348f190b3e001186
+CLOUD_ACCOUNT_ID:   5aabb7531de0240abb000024
+SSH_PROFILE_ID:     5aabb7521de0240abb00000d
+```
+
+You can request for status and see how server is transitioning along service statuses (booting, bootstrapping, operational). Then, after a brief amount of time the final status is reached:
```

Comment:
I think it reads out better as `You can retrieve the current status of the server and see how it transitions along different statuses (booting, `...